api-docs/paths/releases/project-release-file.json

```diff
@@ -116,6 +116,14 @@
         "schema": {
           "type": "string"
         }
+      },
+      {
+        "name": "download",
+        "in": "query",
+        "description": "If this is set to true, then the response payload will be the raw file contents. Otherwise, the response will be the file metadata as JSON.",
+        "schema": {
+          "type": "boolean"
+        }
```

Comment:
Should this parameter be in the GET method? Currently you have it in the PUT method.

Reply:
:facepalm: yes, I've moved it!